cpp/src/rolling/rolling_detail.hpp

```diff
@@ -32,31 +32,43 @@ namespace detail
   template <typename ColumnType, class AggOp, cudf::experimental::aggregation::Kind op, bool is_mean>
   static constexpr bool is_supported()
   {
-    constexpr bool comparable_countable_op =
+    constexpr bool is_comparable_countable_op =
       std::is_same<AggOp, DeviceMin>::value ||
       std::is_same<AggOp, DeviceMax>::value ||
       std::is_same<AggOp, DeviceCount>::value;
 
-    constexpr bool timestamp_mean =
-      is_mean &&
-      std::is_same<AggOp, DeviceSum>::value &&
-      cudf::is_timestamp<ColumnType>();
-
-    constexpr bool is_valid_rolling_agg = !std::is_same<ColumnType, cudf::string_view>::value and
-                                          !std::is_same<ColumnType, cudf::dictionary32>::value and
-                                            ((op == experimental::aggregation::SUM) or
+    constexpr bool is_operation_supported = (op == experimental::aggregation::SUM) or
                                              (op == experimental::aggregation::MIN) or
                                              (op == experimental::aggregation::MAX) or
                                              (op == experimental::aggregation::COUNT) or
-                                             (op == experimental::aggregation::MEAN)) and
-                                           (cudf::is_numeric<ColumnType>() or
-                                            comparable_countable_op or
-                                            timestamp_mean);
+                                             (op == experimental::aggregation::MEAN);
+
+    constexpr bool is_valid_timestamp_agg = cudf::is_timestamp<ColumnType>() and
```

Comment:
Before this change, timestamp supported mean (as it should). This is tested in the gtests, so perhaps this is the source of the failure?